gavel/templates/admin.html

```diff
@@ -31,7 +31,8 @@ <h2>Projects</h2>
           <th class="sort-default" data-sort-method="number" data-sort-order="desc">Mu</th>
           <th data-sort-method="number">Sigma Squared</th>
           <th data-sort-method="number">Votes</th>
-          <th data-sort-method="number">Seen</th> 
+          <th data-sort-method="number">Seen</th>
+          <th data-sort-method="number" data-sort-order="desc">Absent</th>
```

Comment:
Can you remove the changes from this file? The changes here are important enough that I'm going to commit directly to master. Thanks!